### PR TITLE
dcos-608 add message if listing deployment app that doesn't exist

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

**Comment by mgummelt** [comment]
@tamarrow what do you want to do with this?

**Comment by tamarrow** [comment]
@mgummelt want to review it?

**Comment by mgummelt** [comment]
I will once tests pass

**Comment by tamarrow** [comment]
@mgummelt bump

File: cli/dcoscli/marathon/main.py [hunk 1]
```diff
@@ -753,6 +753,9 @@ def _deployment_list(app_id, json_):
 
     deployments = client.get_deployments(app_id)
 
+    if app_id and not deployments:
+        raise DCOSException("There are no deployments for '{}'".format(app_id))
```
Comment (mgummelt) [inline cli/dcoscli/marathon/main.py]: Shouldn't we print something similar if there is no app_id?

Comment (jsancio) [inline cli/dcoscli/marathon/main.py]: +1

Comment (tamarrow) [inline cli/dcoscli/marathon/main.py]: I don't remember why I didn't add anything. I'll do that now.

File: cli/dcoscli/marathon/main.py [hunk 2]
```diff
@@ -761,6 +761,14 @@ def _deployment_list(app_id, json_):
 
     deployments = client.get_deployments(app_id)
 
+    if not deployments:
+        msg = "There are no current deployments"
```
Comment (mgummelt) [inline cli/dcoscli/marathon/main.py]: Remove "current".  It's redundant

File: cli/dcoscli/marathon/main.py [hunk 3]
```diff
@@ -761,6 +761,14 @@ def _deployment_list(app_id, json_):
...
+        if app_id:
+            msg += " for '{}'".format(app_id)
+        if json_:
```
Comment (mgummelt) [inline cli/dcoscli/marathon/main.py]: we don't print exceptions in json format

File: cli/dcoscli/marathon/main.py (hunk 2 above)
Comment (tamarrow) [inline cli/dcoscli/marathon/main.py]: done

File: cli/dcoscli/marathon/main.py (hunk 3 above)
Comment (tamarrow) [inline cli/dcoscli/marathon/main.py]: This is the same thing from [pr 242](https://github.com/mesosphere/dcos-cli/pull/242#discussion_r34212844)

Comment (mgummelt) [inline cli/dcoscli/marathon/main.py]: It is.  And in both cases we want to print to stderr w/o JSON formatting

**Comment by tamarrow** [comment]
this is fixed now

**Comment by mgummelt** [comment]
LGTM

**Pull request merged**